scripts/build/termux_create_pacman_subpackages.sh

```diff
@@ -1,27 +1,21 @@
 termux_create_pacman_subpackages() {
-	# Sub packages:
-	local _ADD_PREFIX=""
-	if [[ "$TERMUX_PACKAGE_LIBRARY" == 'glibc' ]]; then
-		_ADD_PREFIX="glibc/"
-	fi
-	if [[ "$TERMUX_PKG_NO_STATICSPLIT" == 'false' && -n "$(shopt -s globstar; shopt -s nullglob; echo ${_ADD_PREFIX}lib/**/*.a)" ]]; then
-		# Add virtual -static sub package if there are include files:
-		local _STATIC_SUBPACKAGE_FILE=$TERMUX_PKG_TMPDIR/${TERMUX_PKG_NAME}-static.subpackage.sh
-		echo TERMUX_SUBPKG_INCLUDE=\"$(find ${_ADD_PREFIX}lib -name '*.a' -o -name '*.la') $TERMUX_PKG_STATICSPLIT_EXTRA_PATTERNS\" > "$_STATIC_SUBPACKAGE_FILE"
-		echo "TERMUX_SUBPKG_DESCRIPTION=\"Static libraries for ${TERMUX_PKG_NAME}\"" >> "$_STATIC_SUBPACKAGE_FILE"
-	fi
+	local TERMUX_PKG_FILE TERMUX_PKG_ARCH TERMUX_PACMAN_PACKAGE_COMPRESS_CMD TERMUX_PARENT_DEPEND_ON_SUBPKG
 
 	# Now build all sub packages
 	rm -Rf "$TERMUX_TOPDIR/$TERMUX_PKG_NAME/subpackages"
-	for subpackage in $TERMUX_PKG_BUILDER_DIR/*.subpackage.sh $TERMUX_PKG_TMPDIR/*subpackage.sh; do
-		[[ -f "$subpackage" ]] || continue
+	for subpackage in "${TERMUX_PKG_SUBPACKAGES_LIST[@]}"; do
+		if [ ! -f "$subpackage" ]; then
```

Comment:
Same as in the Apt case.
```suggestion
		[[ -f "$subpackage" ]] || termux_error_exit "Failed to find subpackage build file \"$subpackage\" of package \"$TERMUX_PKG_NAME\""
```